types/handsontable/index.d.ts

```diff
@@ -502,18 +503,10 @@ declare var Handsontable: {
     utils: ht.Utils;
     renderers: ht.Renderers;
     editors: ht.Editors;
-};
+}
 
 declare module "handsontable" {
```

Comment:
A better way is to use the new `export as namespace` syntax. `declare module "handsontable"` should no longer be used.

```ts
export = Handsontable; // supports `import * as Handsontable from 'handsontable';
export as namespace Handsontable; // support global usage of Handsontable object.
```

Then you can delete the `declare module "handsontable"` and its contents.

1. The top-level namespace also needs to change its name from `ht` to `Handsontable` in order to provide all the types it contains.
2. References to `ht.x` have to change to `Handsontable.x`.